tests/shell/tests/01-run.py

```diff
@@ -45,16 +45,17 @@
 PROMPT = '> '
 
 CMDS = (
-    # test start
     ('start_test', '[TEST_START]'),
+
+    # test empty line input
     ('\n', PROMPT),
 
     # test simple word separation
     ('echo a string', '"echo""a""string"'),
     ('echo   multiple   spaces   between   argv', '"echo""multiple""spaces""between""argv"'),
     ('echo \t tabs\t\t processed \t\tlike\t \t\tspaces', '"echo""tabs""processed""like""spaces"'),
 
-    # test long line
+    # test unknown commands
```

Comment:
This change is unrelated and the initial comment was correct (the purpose of the test line 58 is to test long lines). But the new comment could be kept and moved before line 62 (since it makes sense for this one).

Reply:
It's not really testing long lines here, because these test strings have a fixed length. The test done by calling `check_line_exceeded` is better suited for this purpose, because it makes use of the shell's buffer size automatically and writes characters beyond the buffer size. Maybe we should replace the long sequence of numbers by a shorter one or by something completely different instead?

Reply:
> The test done by calling check_line_exceeded is better suited for this purpose, because it makes use of the shell's buffer size automatically and writes characters beyond the buffer size.

It this case is already covered by another test then I would suggest to remove this case and only keep 'unknown_command' one. @fjmolinas what do you think ?

Reply:
Well its still an unrelated change, even if it makes sense, maybe you could split those into a separate commit @HendrikVE ?

Reply:
It's in a separate commit now